bos

```diff
@@ -1131,6 +1134,7 @@ prog
   .option('--node <node_name>', 'Saved node to open channels')
   .option('--opening-node <node_name>', 'Open with saved node', REPEATABLE)
   .option('--set-fee-rate <ppm>', 'Set forward fee rate to peer', REPEATABLE)
+  .option('--trusted <is_trusted_funding>', 'Open a trusted channel (true/false)', REPEATABLE)
```

Comment:
maybe type should be expanded instead of adding a new flag? like pubilc-trusted and private-trusted?

Reply:
You mean the type flag should have 4 options?

Reply:
Yeah I think 2 more options is ok

Reply:
Cool. Ok.